src/Mvc/Mvc.RazorPages/src/PageLoggerExtensions.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable disable warnings
```

Comment:
?

Reply:
Too many warnings to deal with. It's mostly logging code so I'm skipping over it for now.